.github/workflows/ci.yml

```diff
@@ -166,3 +180,168 @@ jobs:
           sudo pkg install -y cmake ninja sdl2
           cmake -B build -G Ninja .
           cmake --build build --verbose -- -j`sysctl -n hw.ncpu`
+
+  linux-mingw:
```

Comment:
Do we want to replace the Windows MinGW build with this? The only difference should be the host OS, so let's use whichever one is faster and drop the other.